eva-accession-core/src/main/java/uk/ac/ebi/eva/accession/core/contigalias/ContigAliasNaming.java

```diff
@@ -0,0 +1,32 @@
+/*
+ *
+ * Copyright 2022 EMBL - European Bioinformatics Institute
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+package uk.ac.ebi.eva.accession.core.contigalias;
+
+/**
+ * This enum can be used to specify one of the synonyms supported by the contig alias service
+ */
+public enum ContigAliasNaming {
+    GENBANK_SEQUENCE_NAME,
+    ENA_SEQUENCE_NAME,
+    INSDC,  // this is the same as GenBank,
+    REFSEQ,
+    UCSC,
+    MD5_CHECKSUM,
+    TRUNC512_CHECKSUM,
+    NO_REPLACEMENT  // do not use any particular naming, just keep whatever contig is provided
```

Comment:
Is there a way to have these term being lower cased and dot (.) separated? 
I'm trying to follow the standard from [identifiers.org|https://registry.identifiers.org/registry]

Reply:
@tcezard  While it is not possible to have the Java identifiers conform to the convention, we can follow [this method](https://stackoverflow.com/a/3978690) to attach string labels to the Java identifiers. These string labels can be made to conform to the convention that you suggest.

Reply:
While we're talking about names, I'm not totally sold on the API parameter name `contigAliasNaming` unless it's standard terminology.  Something like `contigNamingConvention` seems more natural to me.

(This is purely from the user's perspective, internally we can call it whatever we like.)

Reply:
I did what @sundarvenkata-EBI suggested for the string labels, but I don't think it actually affects what the parameter value is in the API as Spring seems to use the name rather than the value of enum... :roll_eyes:  I tried a few other things including using a custom converter and didn't get that to work either, so I've left it for now.

Worth pointing out we don't do this for other enums in our API, e.g. choice of variant type, so at least we're consistent!